proposals/0065-svm-fraud-proofs.md

```diff
@@ -0,0 +1,102 @@
+---
+simd: '0065'
+title: Fraud Proofs for SVM
+authors:
+  - Anatoly Yakovenko
+category: Standard/Meta
+type: Core
+status: Draft
+created: 2023-8-9
+feature: (fill in with feature tracking issues once accepted)
+---
+
+## Summary
+
+This SIMD proposes a design for adding succinct proofs of invalid
+computation to Solana's SVM. This would make it much easier to
+verify the correctness of transactions, and to detect fraud or
+malicious activity.
+
+
+## Motivation
+
+With fraud proofs, honest participants in the network will be able
+to construct and gossip a fraud proof that can be easily verified
```

Comment:
Fraud proofs are generally generated when there is an objection on the transaction on L2s which halts that specific block and both the block producer & objector stake some assets then go in proof showdown mode, It makes sense on L2s as their tps is still low and sequencers are the only ones posting proofs. 
--- 
**How would something like this work on Solana and its effects on the wider network?**
Would there be a fork while most of the general ops continue?